src/core/core.animations.js

```diff
@@ -42,6 +42,17 @@ function copyOptions(target, values) {
 	delete values.options;
 }
 
+function extensibleConfig(animations) {
```

Comment:
It might be worth adding a comment here as to why this function is necessary. like is there a known option with object value that you want to ignore?

Reply:
We only want non objects, like 'delay' and such. Might be easier to whitelist the ones that need to be inherited. I've been planning to revisit this, when the time comes :)

Reply:
Yeah, that made sense to me. I just wasn't sure what object values were present that made it necessary to filter to end up with only non-objects